api/filter/schema.go

```diff
@@ -136,6 +144,25 @@ func handleSpecialField(resource string, reqBody []byte) ([]byte, error) {
 	return reqBody, nil
 }
 
+func handleDefaultValue(resource string, reqBody []byte) ([]byte, error) {
+	// go jsonschema lib doesn't support setting default values, so we need to set for some fields necessary
+	if resource == "routes" {
+		var route map[string]interface{}
+		err := json.Unmarshal(reqBody, &route)
+		if err != nil {
+			return reqBody, fmt.Errorf("read request body failed: %s", err)
+		}
+		if _, ok := route["status"]; !ok {
```

Comment:
Just to make sure, 
1. `!ok` will be truthy when `ok` is `falsy`, If one route is not enabled, then after this filter, the status will be?
2. The `status` now is enabled by default after created, if that route has something wrong, we have to offline it after published? 🤔 I'm not sure if it's ok.

Reply:
`if _, ok := route["status"]; !ok {` is checking whether there is a `status` field in the route config, if not, use the default value, otherwise use the submitted `status` value.

Reply:
Got it! Another question is should we enable it by default? If there has something wrong with that route, we have to offline it 🤔

Reply:
Q3: Could we debug an offline route in `OnlineDebug` feature? I'm not sure if `OnlineDebug` means debug online routes 😂

Reply:
> 2. The `status` now is enabled by default after created, if that route has something wrong, we have to offline it after published? 🤔 I'm not sure if it's ok.

1. we will pass `status: 1` when create a route via fe, if user did not disable this field
2. create route without `status`,  golang will treat this field as 0, which goes against the original intent defined in APISIX so @nic-chen add these code to solve this problem.

Reply:
Got it, thanks!

Reply:
> feature

we can debug a online route with `APISIX_IP:PORT/url` via Online debug function
and we can debug a offline route with `upstream_IP:PORT/url` via Online debug function